soc/nxp/imx/imx9/Kconfig.soc

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022,2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_IMX9
+	bool
+	select SOC_FAMILY_IMX
+
+config SOC_MIMX9352
+	bool
+	select SOC_SERIES_IMX9
+
+config SOC_MIMX9352_A55
+	bool
+	select SOC_MIMX9352
+	help
+		NXP i.MX93 A55
```

Comment:
help text indent it 1x tab followed by 2x spaces

Reply:
fixed, thanks.